### PR TITLE
after major rewrite

### PR DESCRIPTION
**Pull request by djdanielsson**
all the other little stuff

### CODE REVIEW

File: roles/gateway_applications/README.md [hunk 1]
```diff
@@ -39,13 +128,26 @@ Options for the `applications_list` variable:
...
 | `state`                     |      `present`      |    no    | str  | Desired state of the application.                                                      |
+>>>>>>> 40b40ddac1c00aac7d878bd41af23a6d562296e5
 
 ### Standard Application Data Structure
 
 #### Json Example
 
 ```json
  {
+<<<<<<< HEAD
```
Comment (sean-m-sullivan) [inline roles/gateway_applications/README.md]: Merge Conflict heads here

File: roles/dispatch/tests/test.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+<<<<<<< HEAD
```
Comment (sean-m-sullivan) [inline roles/dispatch/tests/test.yml]: More head fails

**Comment by sean-m-sullivan** [comment]
Found 16 <<<<<<< HEAD in various files.

**Comment by djdanielsson** [comment]
good catch, I missed those test files. should be clear now, we will probably need to rewrite a lot of the tests if not all of them.

**Review by sean-m-sullivan — APPROVED** [review]

**Comment by felixkrohn** [comment]
I'm a bit surprised the roles `controller_dispatch`, `controller_filetree_create` and `controller_object_diff` have been deleted with this PR. Is it planned to replace them by something else, should that code be maintained outside of this tree in future or are there any other reasonings I fail to understand?

**Comment by Tompage1994** [comment]
Hi, 

Note that this collection is going through a redesign. All the AAP components are being bought into this collection and some roles are being removed and placed into here: https://github.com/redhat-cop/aap_configuration_extended

Further communications will follow on this issue once the work has been done to change everything

**Pull request merged**